bundle/rhpam/metadata/annotations.yaml

```diff
@@ -4,8 +4,8 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: rhpam-kogito-operator
```

Comment:
should we update that one ?

Reply:
It makes sense and it seems safe: I'm doing it